### DISCUSSION TITLE
e7913d64-cd6d-4115-ae75-5983043ab0d8

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#21] [General]
# e7913d64-cd6d-4115-ae75-5983043ab0d8

아두이노도 매터 한다!!!!! 눈 깜짝할 사이에 출시되버린 Arduino의 공식 Matter 개발보드를 살펴보고 예제를 테스트해봤습니다.

https://www.varofla.com/e7913d64-cd6d-4115-ae75-5983043ab0d8

**Comment by codernoah404** [comment]
좋은 자료, 경험 공유해주셔서 감사합니다!
